packages/melos/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ repository: https://github.com/invertase/melos/tree/main/packages/melos
 issue_tracker: https://github.com/invertase/melos/issues
 
 environment:
-  sdk: '>=2.18.0 <3.0.0'
+  sdk: ">=2.18.0 <3.0.0"
```

Comment:
The standard seems to be that these ranges should be with single quotes, check the output of a newly created flutter project for example.
So all pubspec changes should be removed from this PR too.